app/screens/galoy-address-screen/address-explainer-modal.tsx

```diff
@@ -36,7 +37,7 @@ const styles = EStyleSheet.create({
     shadowRadius: 4,
     elevation: 5,
     width: "90%",
-    marginTop: Platform.OS === "android" ? StatusBar.currentHeight : 40,
+    // marginTop: Platform.OS === "android" ? StatusBar.currentHeight : 40,
```

Comment:
Thanks for your contribution @siiddhantt 
Can you remove the commented code since you've solved the issue?
```suggestion
    
```

Reply:
Yeah sure.